lib/aca_entities/protocols/transmittable/transaction.rb

```diff
@@ -26,6 +26,11 @@ class Transaction < Dry::Struct
         # @return [String]
         attribute :description, Types::String.optional.meta(omittable: true)
 
+        # @!attribute [r] transaction_id
+        # An unambiguous reference to the resource kind
```

Comment:
should we update this comment maybe?

Reply:
We want to use this as a reference for the source so it is good.